component/client/RoomService.js

```diff
@@ -0,0 +1,33 @@
+import * as firebase from "firebase";
+
+export default class RoomService {
```

Comment:
I was saying more of 
```
import * as firebase from "firebase";

class RoomService { ... }

export default RoomService;
```
Since our react components are of the form:
```
import stuff

class Something extends React.Component { ... }

export default Something;
````